src/Microsoft.AspNetCore.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -112,8 +123,7 @@ public void OnResourceExecuted(ResourceExecutedContext context)
         /// <inheritdoc />
         public bool Accept(ActionConstraintContext context)
         {
-            // If this constraint is not closest to the action, it will be skipped.
```

Comment:
Many of the removed comments remain valid and help the reader. Why are they being deleted?